contracts/bondingcurve/BondingCurve.sol

```diff
@@ -150,6 +150,24 @@ contract BondingCurve is IERC1363Spender, Owned {
         emit Sell(recepient, amountA, amountB);
     }
 
+    // Calculate price
+
+    function getBuyPrice(uint128 amountA) public view returns (uint256 price) {
```

Comment:
I wonder if it would be better to name the return value `amountB` instead.

Reply:
done